METRICS.md

```diff
@@ -81,13 +81,13 @@ Here is an example of Grafana dashboard importing.
 |        RunningApplicationCount         |      master       |                                The count of running applications in the cluster.                                |
 |             OfferSlotsTime             |      master       |                                            The time of offer slots.                                             |
 |             PartitionSize              |      master       |          The estimated partition size of last 20 flush window whose length is 15 seconds by defaults.           |
-|            PartitionWritten            |      master       |                                            The active shuffle size.                                             |
-|           PartitionFileCount           |      master       |                                       The active shuffle partition count.                                       |
+|         RegisteredShuffleCount         | master and worker |                                  The value means count of registered shuffle.                                   |
+|           ActiveShuffleSize            | master and worker |                                            The active shuffle size.                                             |
```

Comment:
It is necessary to explain the difference between the two metrics calculation methods of master and worker.